.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: dart
+sudo: false
+
+dart:
+  - dev
+  - stable
+
+dart_task:
+  - test
+  - dartfmt
+  - dartanalyzer
+
+matrix:
+  allow_failures:
+  - dart_task: dartanalyzer
```

Comment:
Because there are still files that break analysis – specifically the multi_file things in the lib directory that aren't cleaned up yet